README.md

```diff
@@ -30,36 +30,37 @@ gem 'cfonb'
-| REF         | `operation_reference`                                                                   | Bank operation reference                                             |
-| IPY         | `debtor_identifier`, `debtor_identifier_type`                                           | Debtor identifier and debtor identifier type                         |
-| IBE         | `creditor_identifier`, `creditor_identifier_type`                                       | Creditor identifier and the type of identifier                       |
-| NPO         | `ultimate_debtor`                                                                       | Name of the ultimate debtor or beneficiary                           |
-| NBU         | `ultimate_creditor`                                                                     | Name of the ultimate creditor or payer                               |
+| Detail Code | Attributes                                                                              | Description                                                                                                                                 |
+| ----------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| FEE         | `fee`, `fee_currency`                                                                   | Represents operation fees the bank is applying                                                                                              |
+| LCC         | `unstructured_label`                                                                    | Not structured label line 1 (first 70 characters)                                                                                           |
+| LC2         | `unstructured_label_2`                                                                  | Not structured label line 2 (last 70 characters)                                                                                            |
+| LCS         | `structured_label`                                                                      | Structured label                                                                                                                            |
+| LIB         | `free_label`                                                                            | Free label                                                                                                                                  |
+| MMO         | `original_currency`, `original_amount`, `scale`, `exchange_rate`, `exchange_rate_scale` | Amount and currency if it has been converted from a foreign currency. The `original_amount` is unsigned, meaning it is always non-negative. |
```

Comment:
can you please make sure all codes have the right attributes implemented ? here for example I see we are missing some stuff